sdk/python/feast/infra/registry/http.py

```diff
@@ -77,6 +79,7 @@ def __init__(
         timeout = httpx.Timeout(5.0, connect=60.0)
         transport = httpx.HTTPTransport(retries=3, verify=False)
         self.base_url = registry_config.path
+        self.client_id = registry_config.clint_id
```

Comment:
Update self.http_client with the client_id. That should suffice the need. No need to pass from each function.